### PR TITLE
allow singular they

### PR DESCRIPTION
**Pull request by joepetrowski**

### CODE REVIEW

File: style-guide.md [hunk 1]
```diff
@@ -123,28 +123,15 @@ Readers can often infer when a "which" clause is restrictive versus unrestrictiv
 
 ### Subject and Pronoun Disagreement
 
-Ensure that pronouns and possessives match the subject they are referring to.
+Singular "they" is acceptable as a pronoun. Make sure that there is no possibility of confusion about to what the pronoun refers (subject vs. object).
```
Comment (Robbepop) [inline style-guide.md]:
```suggestion
Singular "they" is accepted as a pronoun. Make sure that there is no possibility of confusion about to what the pronoun refers (subject vs. object).
```

Comment (Robbepop) [reply style-guide.md]: imo sounds better but not too important

**Comment by Robbepop** [comment]
I ll close my PR (https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/pull/128) in favor of yours.

File: style-guide.md (hunk 1 above)
Comment (joepetrowski) [reply style-guide.md]: Just changed to be active instead of passive, as I should  have done in the first place.

Comment (Robbepop) [reply style-guide.md]: Yep! Sounds really good now.

**Pull request merged**